invenio_circulation/api.py

```diff
@@ -82,5 +89,33 @@ def is_item_available(item_pid):
 
 def get_pending_loans_for_item(item_pid):
     """."""
+    return Loan.get_loans_for_pid(item_pid=item_pid,
+                                   filter_states=['PENDING'])
+
+
+def get_pending_loans_for_document(document_pid):
+    """."""
+    return Loan.get_loans_for_pid(document_pid=document_pid,
+                                   filter_states=['PENDING'])
+
+
+def get_available_item_for_document(document_pid):
+    """Returns an item pid available for this document."""
     # TODO: implement search on ES and fetch results from DB
-    return []
+    for item_pid in get_items_for_document(document_pid):
+        if is_item_available(item_pid):
+            return item_pid
+    return None
+
+def get_items_for_document(document_pid):
```

Comment:
I suggest we follow the the existing invenio conventions everywhere, `get_items_by_doc_pid`